src/www/system_general.php

```diff
@@ -229,6 +232,13 @@
         system_timezone_configure();
         filter_configure();
 
+        if ($timezone_changed) {
+            // force real radvd restart
+            $radvd = find_service_by_name('radvd');
+            killbypid($radvd['pidfile'], 'TERM', true);
+            services_radvd_configure();
```

Comment:
Yes, I'll adapt radvd to fully restart to simplify this code bit.

But keep in mind that time zone changes probably affect a lot more services, some can't even cope with time going backwards so the safest bet is to reboot.